content/blog/announcing_talent_sprint_as_silver_sponsor.md

```diff
@@ -0,0 +1,15 @@
+Title: Announcing TalentSprint as Silver Sponsor
```

Comment:
We can leave this blog for now and update this in #191

Reply:
Ram wanted me to have this blog, so let it be there since #191 would take time as other info is pending.

Reply:
Ok sure